diff --git a/tdesktop/Telegram/SourceFiles/core/launcher.cpp b/tdesktop/Telegram/SourceFiles/core/launcher.cpp
index c059159..ffb91d7 100644
--- a/Telegram/SourceFiles/core/launcher.cpp
+++ b/Telegram/SourceFiles/core/launcher.cpp
@@ -63,6 +63,10 @@ int Launcher::exec() {
 	Logs::start(this); // must be started before Platform is started
 	Platform::start(); // must be started before QApplication is created
 
+	// ALT build: hack to find Qt5 plugins
+	QCoreApplication::addLibraryPath("/usr/lib64/qt5/plugins");
+	QCoreApplication::addLibraryPath("/usr/lib/qt5/plugins");
+
 	auto result = 0;
 	{
 		Application app(this, _argc, _argv);
