diff --git a/tdesktop/Telegram/SourceFiles/platform/linux/specific_linux.cpp b/tdesktop/Telegram/SourceFiles/platform/linux/specific_linux.cpp
index 20f20a5..a6dfb7e 100644
--- a/Telegram/SourceFiles/platform/linux/specific_linux.cpp
+++ b/Telegram/SourceFiles/platform/linux/specific_linux.cpp
@@ -35,6 +35,8 @@ Copyright (c) 2014-2017 John Preston, https://desktop.telegram.org
 
 #include <iostream>
 
+#include <QLocale>
+
 #include <qpa/qplatformnativeinterface.h>
 
 using namespace Platform;
@@ -393,11 +393,13 @@ bool TranslucentWindowsSupported(QPoint globalPosition) {
 }
 
 QString SystemCountry() {
-	return QString();
+	QLocale locale;
+	return QLocale::countryToString(locale.country());
 }
 
 QString SystemLanguage() {
-	return QString();
+	QLocale locale;
+	return locale.name();;
 }
 
 namespace ThirdParty {
