diff --git a/tdesktop/Telegram/SourceFiles/platform/linux/info_linux.cpp b/tdesktop/Telegram/SourceFiles/platform/linux/info_linux.cpp
index 2e2d2af..121f174 100644
--- a/Telegram/SourceFiles/platform/linux/info_linux.cpp
+++ b/Telegram/SourceFiles/platform/linux/info_linux.cpp
@@ -7,6 +7,8 @@ https://github.com/telegramdesktop/tdesktop/blob/master/LEGAL
 */
 #include "platform/linux/info_linux.h"
 
+#include <QLocale>
+
 namespace Platform {
 
 QString DeviceModelPretty() {
@@ -25,11 +27,13 @@ QString SystemVersionPretty() {
 }
 
 QString SystemCountry() {
-	return QString();
+	QLocale locale;
+	return QLocale::countryToString(locale.country());
 }
 
 QString SystemLanguage() {
-	return QString();
+	QLocale locale;
+	return locale.name();
 }
 
 QDate WhenSystemBecomesOutdated() {
