diff --git a/tdesktop/Telegram/SourceFiles/storage/cache/storage_cache_database_object.h b/tdesktop/Telegram/SourceFiles/storage/cache/storage_cache_database_obje
index 8f1ea3c..eba34f9 100644
--- a/tdesktop/Telegram/SourceFiles/storage/cache/storage_cache_database_object.h
+++ b/tdesktop/Telegram/SourceFiles/storage/cache/storage_cache_database_object.h
@@ -12,6 +12,7 @@ https://github.com/telegramdesktop/tdesktop/blob/master/LEGAL
 #include "base/binary_guard.h"
 #include "base/concurrent_timer.h"
 #include "base/bytes.h"
+#include "base/openssl_help.h"
 #include "base/flat_set.h"
 #include <set>
 #include <rpl/event_stream.h>
